deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -222,29 +223,19 @@ declare(Q, _Node) when ?amqqueue_is_quorum(Q) ->
             Err
     end.
 
-start_cluster(Q) ->
+internal_declare(Q, RaName, LeaderNode, FollowerNodes) ->
```

Comment:
I would put `internal_declare` after `start_cluster` in the file (old school standard erlang).